App/www/index.html

```diff
@@ -175,18 +175,18 @@
             </div>
         </div>
     </div>
-    
-    </div>      
 
-          
     </div>
-            
-                    
+
+
+    </div>
+
+
     <script type="text/javascript" src="cordova.js"></script>
     <script type="text/javascript" src="lib/framework7/js/framework7.bundle.min.js"></script>
     <script type="text/javascript" src="js/jquery-3.4.1.min.js"></script>
     <script type="text/javascript" src="js/moments.js"></script>
-    <script type="text/javascript" src="js/push.js"></script>
```

Comment:
Hi Bruno, I am unsure about this proposed change too. Let me see what happens when we got the emulator working correctly :) Thanks.

Reply:
Oh, sorry, I was playing with this branch and this slipped in. Let me remove this line.